syntax = "proto3";

package your_package_name;

service ModelService {
  rpc TrainModel (TrainModelRequest) returns (TrainModelResponse);
}

message TrainModelRequest {
  string model_name = 1;
  float parameter1 = 2;
  float parameter2 = 3;
  repeated float features = 4;
  repeated float labels = 5;
}

message TrainModelResponse {
  string model_name = 1;
  string status = 2;
}
